src/chapters/14-provider-components.md

```diff
@@ -350,11 +346,11 @@ Hooray, it works! Awesome. Now that we've tried this out manually in the UI, let
  });
 ```
 
-Great! In the process of adding this test, we'll notice that we also extracted our setup (`setProperties`) into the before hooks. We also added used the `fillIn` test helper in our newly-added test.
+Great! In the process of adding this test, we'll notice that we also extracted our setup (`setProperties`) into the before hooks. We also used the `fillIn` test helper in our newly-added test.
 
 > Zoey says...
 >
-> This search fucntionality is not perfect. Ideally, it would also be case-insensitive and allow you to search by city, category, type, or description. If you're looking for a challenge, see if you can improve on our search!
+> This search functionality is not perfect. Ideally, it would also be case-insensitive and allow you to search by city, category, type, or description. If you're looking for a challenge, see if you can improve on our search!
```

Comment:
```suggestion
> This search functionality is not perfect. Ideally, it would also be case-insensitive, and also allow you to search by city, category, type, or description. If you're looking for a challenge, see if you can improve on our search!
```